File: LTHAM/120/120.cpp
#include <iostream>
using namespace std;

float SoHang(int);

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	cout << "So hang thu " << n << " : " << SoHang(n) << endl;
	return 0;
}

float SoHang(int n)
{
	float ahh = 0;
	float at = 2;
	int i = 2;
	while (i <= n)
	{
		ahh = 5 * at + sqrt(24 * at * at - 8);
		i++;
		at = ahh;
	}
	return ahh;
}

File: CTDK_Python/120/_120.py
from math import sqrt


n = int(input("Nhap n: "))
if (n == 1):
    ahh = 2
else:
    at = 2
    i = 2

    while (i <= n):
        ahh = 5 * at + sqrt(24 * at * at - 8)
        i += 1
        at = ahh

print("Ket qua:", ahh)
